### ISSUE TITLE
Ohai 8.1 fails with error in Digital Ocean plugin

### ISSUE THREAD

**Issue by tas50** [chef/ohai#487]
OS X 10.10.2
Ruby 2.2

```
[2015-02-17T11:59:33-08:00] WARN: Plugin Method Error: </Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/plugins/digital_ocean.rb>: unsupported operation 'IPAddress'
[2015-02-17T11:59:36-08:00] ERROR: Encountered error while running plugins: #<Ohai::Exceptions::AttributeNotFound: No such attribute: 'digital_ocean'>
/Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/provides_map.rb:97:in `block in find_closest_providers_for': No such attribute: 'digital_ocean' (Ohai::Exceptions::AttributeNotFound)
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/provides_map.rb:95:in `each'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/provides_map.rb:95:in `find_closest_providers_for'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/runner.rb:90:in `fetch_plugins'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/runner.rb:73:in `run_v7_plugin'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/runner.rb:43:in `run_plugin'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/system.rb:94:in `block in run_plugins'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/system.rb:93:in `each'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/system.rb:93:in `run_plugins'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/system.rb:71:in `all_plugins'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/application.rb:90:in `run_application'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/lib/ohai/application.rb:70:in `run'
	from /Users/tim/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/ohai-8.1.0/bin/ohai:42:in `<top (required)>'
	from /Users/tim/.rbenv/versions/2.2.0/bin/ohai:23:in `load'
	from /Users/tim/.rbenv/versions/2.2.0/bin/ohai:23:in `<main>'
```

**Comment by jdmundrawala** [comment]
https://github.com/chef/ohai/pull/489

**Comment by jdmundrawala** [comment]
Released Ohai 8.1.1
